### ISSUE TITLE
NOCIF_dram vs NOCIF_sram

### ISSUE THREAD

**Issue by quasar-s** [nvdla/hw#149]
Why `NV_NVDLA_NOCIF_dram` is replaced with `NV_NVDLA_NOCIF_sram` in v2? 
What would be the internal differences between the two modules? 

I'm asking because implementation of `NV_NVDLA_NOCIF_sram` is missing in V2 (but it's instantiated), so I replaced it with `NV_NVDLA_NOCIF_dram`.

**Comment by nvdsmith** [comment]
Right.  The existing nv_small config doesn't have the sram interface enabled.  So it's not in the RTL.  However, the next release of the configurable RTL will add configurations with the sram interface enabled.

**Comment by quasar-s** [comment]
Thanks  @nvdsmith . Any ideas when the next version is planned to be released?
